src/main/java/com/paneedah/mwc/creativetab/PropsTab.java

```diff
@@ -14,6 +14,6 @@ public PropsTab(int par1, String par2Str) {
 	@Override
 	@SideOnly(Side.CLIENT)
 	public ItemStack createIcon() {
-		return new ItemStack(Guns.M4A1);
+		return new ItemStack(Guns.AK12);
```

Comment:
Could you change it to a prop icon, I think it would make more sense, like any of the barrels maybe?